### ISSUE TITLE
false positive for UnusedPrivateElements writeOnlyProperty

### ISSUE THREAD

**Issue by meridius** [slevomat/coding-standard#69]
There is a false positive for `SlevomatCodingStandard.Classes.UnusedPrivateElements` sniff when private property is used in a double-quoted string. The sniff is then reporting it as unused `writeOnlyProperty`.

```php
<?php

class FalsePositiveWriteOnlyProperty
{
    private $dir;

    public function __construct($name)
    {
        $this->dir = dirname($name);
    }

    protected function buildIconPath($name)
    {
        return "$this->dir/$name.png"; // here the property is used
    }
}
```

**Comment by ondrejmirtes** [comment]
Duplicate of #63. See the discussion there please. Thanks for the report.
